app/models/manageiq/providers/nuage/network_manager/cloud_subnet.rb

```diff
@@ -1,7 +1,7 @@
 class ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet < ::CloudSubnet
   has_many :security_groups, :dependent => :destroy
 
-  include AnsibleCrudMixin
+  include ManageIQ::Providers::Nuage::AnsibleCrudMixin
```

Comment:
@Ladas any idea why the shorthand include works on CloudSubnet, but not on FloatingIp/SecurityGroup? There it attempts to include it with full namespace like 

```
ManageIQ::Providers::Nuage::NetworkManager::FloatingIp::AnsibleCrudMixin
```

but here on CloudSubnet it just knows the correct namespace 🤔

Reply:
Think the magic loader tries several variants, using the current scope should be first one (so ManageIQ::Providers::Nuage::NetworkManager::CloudSubnet)

So for me it's always safer to just specify the full path